src/main/java/net/minestom/server/inventory/Inventory.java

```diff
@@ -115,7 +98,7 @@ public String getTitle() {
      *
      * @param title the new inventory title
      */
-    public void setTitle(@NotNull String title) {
```

Comment:
If you're removing these methods you might as well remove the deprecated constructor.